tasks/main.yml

```diff
@@ -25,7 +25,10 @@
   args:
     creates: /etc/php5/cli/conf.d/20-mcrypt.ini
 
-- name: Copy php-fpm configuration
+- name: Copy php-fpm php.ini configuration
+  template: src=php.ini dest=/etc/php5/fpm/
```

Comment:
This won't work. `/etc/php5/fpm/` contains `php5-fpm` configuration files, not `php.ini` files. They're sadly totally different.

`php.ini` should likely go in `/etc/php/config/` (off the top of my head...)